src/highlight.c

```diff
@@ -2948,6 +2995,9 @@ highlight_list_one(int id)
 				    sgp->sg_cterm_bg, NULL, "ctermbg");
     didh = highlight_list_arg(id, didh, LIST_INT,
 				    sgp->sg_cterm_ul, NULL, "ctermul");
+    if (sgp->sg_cterm_font != 0)
+	didh = highlight_list_arg(id, didh, LIST_INT,
+				    sgp->sg_cterm_font + 1, NULL, "ctermfont");
```

Comment:
Since it's different from all the rest, why `+ 1`? Upon closer inspection, it seems related to the tests, but I don't see the relation.

Reply:
The others use 0 as a sentinel value as far as I can tell, while ctermfont has 0 as a valid value. All the others therefore add one to bring the valid values into a certain range, and then subtract it later. I could probably do a similar thing for ctermfont come to think of it.